network/interfaces.go

```diff
@@ -299,4 +301,13 @@ type Gateway interface {
 	GetState() insolar.NetworkState
 	OnPulse(context.Context, insolar.Pulse) error
 	NewGateway(insolar.NetworkState) Gateway
+	Auther() Auther
+}
+
+type Auther interface {
```

Comment:
I think Auther interface and impls should be outside from network package. We should remove the injection of MessageBus